tests/FakeItEasy.Tests/Configuration/AnyCallConfigurationTests.cs

```diff
@@ -50,6 +50,18 @@ public void WithReturnType_should_set_the_type_to_the_configured_rule()
         }
 
         [Test]
+        public void WithNonVoidReturnType_should_set_the_flag_to_apply_to_all_return_types()
+        {
+            // Arrange
+
+            // Act
+            this.configuration.WithNonVoidReturnType();
+
+            // Assert
+            Assert.That(this.callRule.ApplicableToAllNonVoidReturnTypes, Is.EqualTo(true));

```

Comment:
Is this the only reason the property has a getter? I'd be inclined to drop it (and this test) in that case. Of course, that leaves  a set-only property, but we could switch to a method.
On the other hand, the property as is _does_ follow the pattern established by `ApplicableToMembersWithReturnType`, so if it stays as is, I'm okay.